package.json

```diff
@@ -17,7 +18,7 @@
         "start": "node scripts/prepare && start-storybook -p 6006 -s public",
         "dev": "node scripts/prepare && start-storybook -p 6006 -s public --no-manager-cache",
         "storybook": "node scripts/prepare && start-storybook -p 6006 -s public",
-        "build": "node scripts/prepare && vite build",
+        "build": "node scripts/prepare && vite build && cp index.d.ts build",
```

Comment:
The fix is to copy the file into the build folder and include it in the `files` above